realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -312,7 +313,8 @@ void BaseRealSenseNode::setupPublishers()
                 aligned_image_raw << "aligned_depth_to_" << _stream_name[stream] << "/image_raw";
                 aligned_camera_info << "aligned_depth_to_" << _stream_name[stream] << "/camera_info";
 
-                _depth_aligned_image_publishers[stream] = image_transport.advertise(aligned_image_raw.str(), 1);
+                std::shared_ptr<FrequencyDiagnostics> frequency_diagnostics(new FrequencyDiagnostics(_fps[stream], _stream_name[stream], _serial_no));
```

Comment:
Please add the prefix of the aligned frames to the stream name and pass it to `frequency_diagnostics` ctor.
```
std::string aligned_stream_name = "aligned_depth_to_" + _stream_name[stream];
std::shared_ptr<FrequencyDiagnostics> frequency_diagnostics(new FrequencyDiagnostics(_fps[stream], aligned_stream_name, _serial_no));
```

Reply:
Thanks, you are right! Just updated the PR.